src/Microsoft.SqlTools.Migration/MigrationService.cs

```diff
@@ -611,15 +611,33 @@ internal async Task HandleGetArmTemplateRequest(
         {
             try
             {
-                Logger.Verbose("request received in toolsservice");
                 ProvisioningScriptServiceProvider provider = new ProvisioningScriptServiceProvider();
                 string searchPattern = $"*{targetType}-Baseline*.json";
                 string skuRecommendationReportFilePath = Directory.GetFiles(SqlAssessmentConfiguration.ReportsAndLogsRootFolderPath, searchPattern).FirstOrDefault();
-                Logger.Verbose($"Logging report file path -- {skuRecommendationReportFilePath}");
-                List<SkuRecommendationResult> recommendations = ExtractSkuRecommendationReportAction.ExtractSkuRecommendationsFromReport(skuRecommendationReportFilePath);
-                Logger.Verbose($"recommendations generated-- {recommendations.Count}");
+
+                // Save the current Console.Out
+                var originalOut = Console.Out;
+                List<SkuRecommendationResult> recommendations;
+                try
+                {
+                    // Redirect Console.Out to a null stream (no output)
+                    using (var nullWriter = new StreamWriter(Stream.Null))
```

Comment:
Nit: maybe create an issue for this so that you can track work for properly fixing the actual bug.

Reply:
Makes sense will create